searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizer/TypeModelRecognizer.java

```diff
@@ -1,4 +1,4 @@
-package org.jalau.at18.searchobject.modelrecognizer;
+package org.jalau.at18.searchobject.model.objectrecognizer.recognizer;
 
 public enum TypeModelRecognizer {
     YOLO("yolo"),
```

Comment:
You can have "yolo" as a constant.
Example  `public static final String YOLO_STR = "YOLO";`

Reply:
It's a enum so it's not necessary to have "yolo" as a constant